docs/DataModels/DevLakeDomainLayerSchema.md

```diff
@@ -254,44 +251,51 @@ A ref is the abstraction of a branch or tag.
 
-#### refs_commits_diffs
+#### commits_diffs
 
-This table shows the commits added in a new ref compared to an old ref. This table can be used to support tag-based analysis, for instance, '_No. of commits of a tag_', '_No. of merged pull request of a tag_', etc.
+This table shows the commits added in a new commit compared to an old commit. This table can be used to support tag-based and deploy-based analysis.
 
-The records of this table are computed by [RefDiff](https://github.com/apache/incubator-devlake/tree/main/plugins/refdiff) plugin. The computation should be manually triggered after using [GitRepoExtractor](https://github.com/apache/incubator-devlake/tree/main/plugins/gitextractor) to collect commits and refs. The algorithm behind is similar to [this](https://github.com/apache/incubator-devlake/compare/v0.8.0%E2%80%A6v0.9.0).
+The records of this table are computed by [RefDiff](https://github.com/apache/incubator-devlake/tree/main/plugins/refdiff) plugin. The computation should be manually triggered after using [GitRepoExtractor](https://github.com/apache/incubator-devlake/tree/main/plugins/gitextractor) to collect commits. The algorithm behind is similar to [this](https://github.com/apache/incubator-devlake/compare/v0.8.0%E2%80%A6v0.9.0).
 
-| **field**            | **type** | **length** | **description**                                                 | **key**        |
-| :------------------- | :------- | :--------- | :-------------------------------------------------------------- | :------------- |
-| `commit_sha`         | char     | 40         | One of the added commits in the new ref compared to the old ref | FK_commits.sha |
-| `new_ref_id`         | varchar  | 255        | The new ref's id for comparison                                 | FK_refs.id     |
-| `old_ref_id`         | varchar  | 255        | The old ref's id for comparison                                 | FK_refs.id     |
-| `new_ref_commit_sha` | char     | 40         | The commit new ref points to at the time of collection          |                |
-| `old_ref_commit_sha` | char     | 40         | The commit old ref points to at the time of collection          |                |
-| `sorting_index`      | varchar  | 255        | An index for debugging, please skip it                          |                |
+| **field**        | **type** | **length** | **description**                                                       | **key**        |
+| :--------------- | :------- | :--------- | :-------------------------------------------------------------------- | :------------- |
+| `commit_sha`     | char     | 40         | One of the added commits in the new commit compared to the old commit | FK_commits.sha |
+| `new_commit_sha` | char     | 40         | The new commit                                                        |                |

```

Comment:
It looks like `commit_sha` is not the PK of the table. How about moving PK(s) up? Also, don't forget to change the order in the screenshots below.